util/rbac/rbac_test.go

```diff
@@ -128,6 +128,7 @@ p, mike, *, *, foo/obj, deny
 p, trudy, applications, get, foo/obj, allow
 p, trudy, applications/*, get, foo/obj, allow
 p, trudy, applications/secrets, get, foo/obj, deny
+p, danny, applications, get, */obj, allow
```

Comment:
I think it would be good to add some more wildcard patterns to test against, somewhere in the middle etc. Just to make sure :)

Reply:
added one more case with `*` in the middle with two checks.